zhaquirks/tuya/ts0044.py

```diff
@@ -321,3 +321,98 @@ class TuyaSmartRemote0044TOPlus(CustomDevice, Tuya4ButtonTriggers):
             },
         },
     }
+
+class TuyaSmartRemote0044TOPlusVarietyTwo(CustomDevice, Tuya4ButtonTriggers):
```

Comment:
I was a bit unsure of the name to use here...I'm sure it would be easy to improve on my choice; suggestions welcome